### PR TITLE
upgrade @aws-amplify/ui-react

### PR DESCRIPTION
**Pull request by katiegoines**
#### Description of changes:
Upgraded @aws-amplify/ui-react to v6

staging for confirmation: https://upgrade-ui-react.d1egzztxsxq9xz.amplifyapp.com/

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] Swift
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://docs.amplify.aws/)` 
            HTML: `<a href="https://docs.amplify.aws/">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

**Comment by katiegoines** [comment]
@timngyn found that with the upgraded version, some colors aren't rendering correctly. Looking into it!

File: src/styles/feedback.scss [hunk 1]
```diff
@@ -34,7 +34,7 @@
   justify-content: start;
   font-size: 0.9rem;
   color: inherit;
-  border-color: var(--amplify-colors-brand-secondary-);
+  border-color: var(--amplify-colors-secondary-);
```
Comment (hbuchel) [inline src/styles/feedback.scss:37]: This css custom property isn't doing anything so we can remove it, but actually looking at this CSS a little more we could remove all the base values on .vote-button so it becomes:
```
.vote-button {
  .amplify-text {
    padding-left: 7px;
  }
}
```
I think the base font size which is a teeny bit bigger is fine.

**Review by hbuchel — APPROVED** [review]
Just a small nit but otherwise LGTM!

**Review by timngyn — APPROVED** [review]
Tested out the staging site and everything looks good to me!

**Pull request merged**